src/Appwrite/Extend/Exception.php

```diff
@@ -151,6 +151,11 @@ class Exception extends \Exception
     public const PROVIDER_CONTRIBUTION_CONFLICT    = 'provider_contribution_conflict';
     public const GENERAL_PROVIDER_FAILURE          = 'general_provider_failure';
 
+    /** Sites */
+    public const SITE_NOT_FOUND                    = 'site_not_found';
+    public const SITE_FRAMEWORK_UNSUPPORTED        = 'site_framework_unsupported';
+    public const SITE_TEMPLATE_NOT_FOUND           = 'site_template_not_found';
+
     /** Functions */
     public const FUNCTION_NOT_FOUND                = 'function_not_found';
     public const FUNCTION_RUNTIME_UNSUPPORTED      = 'function_runtime_unsupported';
```

Comment:
```suggestion
    public const FUNCTION_RUNTIME_UNSUPPORTED      = 'function_runtime_unsupported';
```

Should this actually be not found?

Reply:
Agree, but we do it in 16 errors already. I dont want to change error types, as that would be breaking change for those doing custom messages for exceptions.
We dont currently have system in-place for exception type backwards compatibility.

I agree to have project to improve this and add backwards compatibility, but separate from Sites G1